core/src/test/scala/unit/kafka/server/ReplicaManagerTest.scala

```diff
@@ -3600,7 +3600,10 @@ class ReplicaManagerTest {
       "clusterId",
       time,
       _ => Optional.of(mockLog),
-      brokerTopicStats)
+      brokerTopicStats,
+      mock(classOf[MetadataCache])
```

Comment:
Could you please extract the `mock` as variable before using it to keep it uniform?

```
val metadataCache =  mock(classOf[MetadataCache])
...
when(metadataCache.getTopicId(topic))...

```

Reply:
Sure. Moved the `mock(classOf[MetadataCache])` as variable now.